lib/commands/gesture.js

```diff
@@ -6,6 +6,16 @@ import log from '../logger';
 
 const helpers = {}, extensions = {}, commands = {};
 
+function toElementId (opts = {}) {
+  const el = opts.elementId || opts.element;
```

Comment:
not an issue, but would it be helpful to print a warn log for opts.element to inform the deprecation?

Reply:
makes sense. Added a notification